src/components/policy/test/policy_manager_impl_test.cc

```diff
@@ -65,11 +68,11 @@ namespace components {
 namespace policy {
 
 class PolicyManagerImplTest : public ::testing::Test {
-protected:
-  PolicyManagerImpl *manager;
-  MockCacheManagerInterface *cache_manager;
+ protected:
+  PolicyManagerImpl* manager;
```

Comment:
@orhan-mehmedov is it possible to store this values by value or reference ot auto pointer or even shared pointer?